zerver/openapi/zulip.yaml

```diff
@@ -6659,7 +6659,7 @@ paths:
         - name: filter_id
           in: path
           description: |
-            The ID of the filter that you want to remove.
+            The ID of the linkifier that you want to remove.
```

Comment:
This could be in a separate commit, because it is unrelated with this new feature.